apps/web/src/features/feedback/ui/feedback-step/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { ComponentProps, MouseEvent, createContext, useContext, useState } from 'react'
+import { FEEDBACK_STEP, FeedbackStepType } from '../../model/feedback-step'
+import { cn } from '@repo/util'
+import { cva } from 'class-variance-authority'
+
+const feedbackStepVariants = cva(
+  'm-auto flex h-[1.125rem] w-[1.125rem] cursor-pointer items-center justify-between rounded-full border-2 border-label-neutral',
+  {
+    variants: {
+      isCurrentStep: {
+        true: 'bg-label-alternative',
+        false: 'bg-background-alternative'
+      }
+    }
+  }
+)
+
+type FeedbackStepProps = ComponentProps<'div'> & {
+  defaultValue?: FeedbackStepType
+}
+
+type FeedbackStepContextType = {
+  currentStep: FeedbackStepType
+  onChange: (value: FeedbackStepType) => void
+}
+
+const ProgressStepContext = createContext<FeedbackStepContextType | null>(null)
+
+function FeedbackStep({ children, defaultValue, ...props }: FeedbackStepProps) {
+  const [currentStep, setCurrentStep] = useState<FeedbackStepType>(
+    defaultValue ?? FEEDBACK_STEP.PART.VALUE
+  )
+
+  return (
+    <ProgressStepContext.Provider value={{ currentStep, onChange: setCurrentStep }}>
+      <div
+        className='flex justify-between'
+        {...props}>
+        {children}
+      </div>
+    </ProgressStepContext.Provider>
+  )
+}
+
+type FeedbackStepItemProps = ComponentProps<'div'> & {
+  stepLabel: string
+  stepValue: FeedbackStepType
+}
+
+function OneStep({ stepLabel, stepValue, className, onClick, ...props }: FeedbackStepItemProps) {
+  const context = useContext(ProgressStepContext)
+
+  if (!context) {
+    throw new Error('FeedbackStepContext must be used within a FeedbackStep')
+  }
+
+  const { currentStep, onChange } = context
+  const isCurrentStep = currentStep === stepValue
+  const handleClickStep = (event: MouseEvent<HTMLDivElement, globalThis.MouseEvent>) => {
+    onChange(stepValue)
+    onClick?.(event)
+  }
+
+  return (
+    <div
+      className='z-10 mx-[-1rem] flex flex-col gap-2'
+      onClick={handleClickStep}>
+      <div className='text-center'>
+        <span className='text-l-normal font-medium text-neutral-30'>{stepLabel}</span>
```

Comment:
p2

해당 부분 Typography 컴포넌트로 변경할 수 있습니다.

```suggestion
<Typography variant='label-normal' fontWeight='medium' color='neutral'>{stepLabel}</Typography>
```

Reply:
[해당 커밋](https://github.com/mobi-production/ani-fe/pull/12/commits/dd7fb29477ccd60f4cfe49f004f537f318462fb1)에서 적용해두었습니당